src/partitioning/block.jl

```diff
@@ -10,17 +10,19 @@ A method for partitioning spatial objects into blocks of given `sides`.
 """
 struct BlockPartition{Dim,T} <: PartitionMethod
   sides::SVector{Dim,T}
+  neighbors::Bool
 end
 
-BlockPartition(sides::NTuple{Dim,T}) where {Dim,T} = BlockPartition{Dim,T}(sides)
-
-BlockPartition(sides::Vararg{T,Dim}) where {Dim,T} = BlockPartition(sides)
+BlockPartition(sides::NTuple{Dim,T}, neighbors = false) where {Dim,T} = BlockPartition{Dim,T}(sides, neighbors)
+BlockPartition(sides::Vararg{T,Dim}; neighbors::Bool = false) where {Dim,T} = BlockPartition(SVector(sides), neighbors)
```

Comment:
```suggestion
BlockPartition(sides::NTuple{Dim,T}; neighbors=false) where {Dim,T} = BlockPartition{Dim,T}(sides, neighbors)
BlockPartition(sides::Vararg{T,Dim}; neighbors=false) where {Dim,T} = BlockPartition(SVector(sides), neighbors)
```
Please follow the code style convention in the code when possible.

Reply:
Fixed!